google_takeout_parser/parse_json.py

```diff
@@ -160,8 +160,8 @@ def _parse_location_history(p: Path) -> Iterator[Res[Location]]:
                 lat=float(loc["latitudeE7"]) / 1e7,
                 dt=_parse_timestamp_key(loc, "timestamp"),
                 accuracy=None if accuracy is None else float(accuracy),
-                deviceTag=None if deviceTag is None else deviceTag,
-                source=None if source is None else str(source)
+                deviceTag=None if deviceTag is None else int(deviceTag),
+                source=None if source is None else source
```

Comment:
I think even if its redundant, I would prefer to have the `str` call here. the explicit converts may be a bit wasteful, but the point is to make sure it works with cachew, which converts each item to a database row, which requires a certain type. it also just means that the type hints are accurate, and if/when google changes the schema things dont blow up immediately.